README.md

```diff
@@ -174,40 +186,37 @@ For convenience, the workflow has been run for 100 events and the output is stor
-     ```
    - Modify the input and output file names accordingly, as well as the number of events.
 
 Then, run the configuration file with `cmsRun`.
 
 To get the DQM files, one would have to run step4 of the 21034.1 workflow with the following modifications:
-   - Modify the `--pileup_input` flag as above, add `--no_exec` to the end of command and then run it.
-   - Modify the output configuration file by including the appropriate PU files, as above, and by changing the input file (the one containing `inDQM` from the previous step) and number of events accordingly.
+   - Add `--no_exec` to the end of command and then run it.
+   - Modify the output configuration file by changing the input file (the one containing `inDQM` from the previous step) and number of events accordingly.
 
 Running the configuration file with `cmsRun`, the output file will have a name starting with `DQM`. The name is the same every time this step runs,
 so it is good practice to rename the file, e.g. to `tracking_Iters01LST.root`.
 The MTV plots can be produced with the command:
 ```bash
 makeTrackValidationPlots.py --extended tracking_Iters01LST.root
 ```
+Comparison plots can be made by including multiple ROOT files as arguments.
 
-**Note:** In case one wants to run step2 as well, similar modifications as in step4 (PU files, `--no_exec` flag and input file/number of events) need to be applied.
+**Note:** In case one wants to run step2 as well, similar modifications as in step4 (`--no_exec` flag and input file/number of events) need to be applied.
```

Comment:
Do we need to include the PU files for step2? Or are they only for step1?

Reply:
We need them. This is my bad, because I though this sentence was referring to step4 (wherePU files are not needed). Let me fix it with a quick commit. Sorry, and thanks for catching this!